src/Symfony/Component/Form/Tests/ResolvedFormTypeTest.php

```diff
@@ -103,12 +99,51 @@ public function testGetOptionsResolver()
         $this->assertEquals($resolvedOptions, $resolver->resolve($givenOptions));
     }
 
-    public function testCreateBuilder()
+    public function testGetOptionsResolver()
     {
-        if (version_compare(\PHPUnit_Runner_Version::id(), '3.7', '<')) {
-            $this->markTestSkipped('This test requires PHPUnit 3.7.');
-        }
+        $test = $this;
+        $i = 0;
+
+        $assertIndexAndAddOption = function ($index, $option, $default) use (&$i, $test) {
+            return function (OptionsResolverInterface $resolver) use (&$i, $test, $index, $option, $default) {
+                /* @var \PHPUnit_Framework_TestCase $test */
+                $test->assertEquals($index, $i, 'Executed at index '.$index);
+
+                ++$i;
+
+                $resolver->setDefaults(array($option => $default));
+            };
+        };
+
+        // First the default options are generated for the super type
+        $this->parentType->expects($this->once())
+            ->method('configureOptions')
+            ->will($this->returnCallback($assertIndexAndAddOption(0, 'a', 'a_default')));
+
+        // The form type itself
+        $this->type->expects($this->once())
+            ->method('configureOptions')
+            ->will($this->returnCallback($assertIndexAndAddOption(1, 'b', 'b_default')));
+
+        // And its extensions
+        $this->extension1->expects($this->once())
+            ->method('configureOptions')
+            ->will($this->returnCallback($assertIndexAndAddOption(2, 'c', 'c_default')));
```

Comment:
``configureOptions`` is called internally by the form type. It is not called by the ResolvedFormType. So it should not appear here